packages/flutter/lib/src/material/chip.dart

```diff
@@ -1821,6 +1821,12 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     assert(debugCheckHasDirectionality(context));
     assert(debugCheckHasMaterialLocalizations(context));
 
+    final EdgeInsetsGeometry _defaultLabelPadding = EdgeInsets.lerp(
```

Comment:
This would be a good place for a comment based on this PR's description:

> Now the chip starts with 8px on each side and as text scaling gets closer to 2 the label padding is linearly interpolated from 8px to 4px. Once the widget has a text scaling of 2 or higher than the label padding remains 4px.

Reply:
Done